megamek/build.gradle

```diff
@@ -50,7 +56,12 @@ application {
 }
 
 ext {
-    mmJvmOptions = ['-Xmx4096m', '--add-opens', 'java.base/java.util=ALL-UNNAMED', '--add-opens', 'java.base/java.util.concurrent=ALL-UNNAMED', '-Dsun.awt.disablegrab=true']
+    mmJvmOptions = [
+        '-Xmx2048m',
```

Comment:
Should this default to 4096m?

Reply:
No, I upped it when I shouldn't have. It should be 2G and not 4G